docs/kyma/docs/037-gs-installation-overrides.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Helm overrides for Kyma installation
+type: Getting Started
+---
+
+Kyma packages its components into [Helm](https://github.com/helm/helm/tree/master/docs) charts that the [Installer](../../../components/installer/README.md) uses.
+This document describes how to configure the Installer with override values for Helm [charts](https://github.com/helm/helm/blob/master/docs/charts.md).
+
+
+## Overview
+
+The Installer is a Kubernetes Operator that uses Helm to install Kyma components.
+Helm provides an *overrides* feature to customize the installation of charts, such as to configure environment-specific values.
+When using Installer for Kyma installation, users can't interact with Helm directly (installation is not an interactive process).
+To customize Kyma installation, Installer exposes a generic mechanism for configuring Helm overrides called **user-defined** overrides.
+
+
+## User-defined overrides
+
+The Installer finds user-defined overrides by reading the ConfigMaps and Secrets deployed in the `kyma-installer` Namespace and marked with the `installer:overrides` Label.
+
+The Installer constructs a single override by inspecting the ConfigMap or Secret entry key name. The key name should be a dot-separated sequence of strings corresponding to the structure of keys in the chart's `values.yaml` file or the entry in chart's template. See the examples below.
+
+Installer merges all overrides recursively into a single YAML stream and passes it to Helm during the Kyma installation/upgrade operation.
+
+
+### Common vs component overides
+
+The Installer looks for available overrides each time a component installation or update operation is due.
+Overrides for the component are composed from two sets: **common** overrides and **component-specific** overrides.
+
+Kyma uses common overrides for the installation of all components. Only ConfigMaps or Secrets marked with the installer:overrides Label contain the definition of overrides (TODO: Fix it).
```

Comment:
The intention of this sentence is that common overrides are labelled only with "installer:overrides" label, no other label is required. I think that this sentence does not quite reflect this condition.

Reply:
Thanks for the clarification. Let's keep what you had before.  Otherwise you can use this:  

Kyma uses common overrides for the installation of all components. Common overrides (ConfigMaps or Secrets) have the label 'installer:overrides' and contain the definition of overrides. They require no additional label.